servicetalk-transport-api/src/main/java/io/servicetalk/transport/api/SpecialConnectExecutionStrategy.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright © 2021 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.servicetalk.transport.api;
+
+/**
+ * package private constants for special Execution
+ */
+enum SpecialConnectExecutionStrategy implements ConnectExecutionStrategy {
```

Comment:
`SpecialConnectExecutionStrategy` -> "Special" as it is used here doesn't quite mean the same thing as it does when it is used in other contexts (e.g. `SpecialHttpExecutionStrategy` has "default" / "never" which are treated "special" all over the code base ... and we plan to remove the special treatment eventually). Consider using different terminology (e.g. `DefaultHttpExecutionStrategy`)